src/library/scala/collection/parallel/mutable/ParArray.scala

```diff
@@ -179,7 +179,7 @@ self =>
 
     override def fold[U >: T](z: U)(op: (U, U) => U): U = foldLeft[U](z)(op)
 
-    override def aggregate[S](z: =>S)(seqop: (S, T) => S, combop: (S, S) => S): S = foldLeft[S](z)(seqop)
+    override def aggregate[B](z: =>B)(seqop: (B, T) => B, combop: (B, B) => B): B = foldLeft[B](z)(seqop)
```

Comment:
could you either take this out, or bring `aggregate` in ParIterableLike.scala in line as well? either way.